packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -239,13 +288,13 @@ export interface ComboBoxProps
    * Provide text to be used in a `<label>` element that is tied to the
    * combobox via ARIA attributes.
    */
-  titleText?: ReactNodeLike;
+  titleText?: ReactNode;
```

Comment:
Here we used to have `ReactNodeLike` but maybe we could change to `string`, since we are using only text.

Reply:
Same thing here, it's `node` down in the proptypes.